src/jvm/test/resources/expected_logs/run_concurrent_test/kotlin_thread_jdk8.txt

```diff
@@ -25,7 +25,7 @@ Detailed trace:
 | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | KotlinThreadRunConcurrentRepresentationTest#1.block(): threw IllegalStateException at BaseRunConcurrentRepresentationTest$testRunWithModelChecker$result$1$1.invoke(RunConcurrentRepresentationTests.kt:40) |                                                                                                                                                                      |
 |   block(): threw IllegalStateException at KotlinThreadRunConcurrentRepresentationTest.block(RunConcurrentRepresentationTests.kt:231)                                                                        |                                                                                                                                                                      |
-|     ThreadsKt.thread(true, false, null, null, -1, block$t$1#1): Thread#2 at KotlinThreadRunConcurrentRepresentationTest.block(RunConcurrentRepresentationTests.kt:241)                                      |                                                                                                                                                                      |
+|     thread { ... } : Thread#2 at KotlinThreadRunConcurrentRepresentationTest.block(RunConcurrentRepresentationTests.kt:241)                                                                                 |                                                                                                                                                                      |
```

Comment:
For now, let's display it as `thread(block$t$1#1)`. 
We will decide how to improve lambdas representation in general later under #519 .